tokio/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.2.15 (April 2, 2020)
+
+### Fixes
+
+- rt: fix queue regression (#2362).
+- sync: Add disarm to `mpsc::Sender` (#2358).
```

Comment:
This could go out shortly later. If it does go out now it should be in an added section.